src/jsdocUtils.js

```diff
@@ -312,11 +316,15 @@ const getPreferredTagName = (
   // e.g. 'tag constructor' for 'constructor':
   // https://github.com/eslint/eslint/issues/13289
   // https://github.com/gajus/eslint-plugin-jsdoc/issues/537
-  const tagPreferenceFixed = _.mapKeys(tagPreference, (_value, key) => {
-    return key.replace('tag ', '');
-  });
+  const tagPreferenceFixed = Object.fromEntries(
+    Object
+      .entries(tagPreference)
+      .map(([key, value]) => {
+        return [key.replace('tag ', ''), value];
```

Comment:
Technically this would be more correct, I didn't include it in the PR to avoid scope creep.

```suggestion
        return [key.replace(/^tag /u, ''), value];
```